hollow/src/main/java/com/netflix/hollow/api/producer/ListenerSupport.java

```diff
@@ -32,14 +36,20 @@
  */
 final class ListenerSupport {
     private final Set<HollowProducerListener> listeners;
+    private final Set<HollowValidationListener> validationListeners;
 
     ListenerSupport() {
         listeners = new CopyOnWriteArraySet<>();
+        validationListeners = new CopyOnWriteArraySet<>();
     }
 
     void add(HollowProducerListener listener) {
         listeners.add(listener);
     }
+    
+    void add(HollowValidationListener listener) {
```

Comment:
Does it need a corresponding remove so it will be balanced similar to HollowProducerListener?